client/memory.go

```diff
@@ -177,3 +182,8 @@ func (c *Memory) SetHTTPRequestSection(section string) Client {
 func (c *Memory) ResetHTTPRequestSection() Client {
 	return c.SetHTTPRequestSection(bucket.SectionRequest)
 }
+
+// Handler returns metrics endpoint for prometheus backend
+func (c *Memory) Handler() http.Handler {
+	return nil
```

Comment:
we'd better return handler always - in this case handler that, for example, returns `405` status code, or return map values encoded in json